homeassistant/components/sensor/command_line.py

```diff
@@ -27,6 +28,7 @@
 
 SCAN_INTERVAL = timedelta(seconds=60)
 
+CONF_JSON_ATTRS = 'json_attributes'
```

Comment:
I would use full name CONF_JSON_ATTRIBUTES

Reply:
Done, have renamed a few other variables using attrs to attributes at the same time.